src/agent/library/skills.js

```diff
@@ -10,19 +10,25 @@ export function log(bot, message, chat=false) {
         bot.chat(message);
 }
 
+export function shouldPlaceTorch(bot) {
```

Comment:
can we move this to world.js? I think that anything the does not change the world state should go there.